src/core/auth/qgsauthcertutils.h

```diff
@@ -373,6 +373,8 @@ class CORE_EXPORT QgsAuthCertUtils
 
   private:
     static void appendDirSegment_( QStringList &dirname, const QString &segment, QString value );
+
+    static QSsl::EncodingFormat sniffEncoding( const QByteArray &payload );
```

Comment:
PR looks good to me.

Just one comment. The word 'sniff' is a bit of coder lingo. Maybe just `encodingFormat(...)` instead? Your call. I don't feel strongly about it.

Reply:
Merging like it is, I'll rename the method with a direct commit to master ...